app/feature/utils.py

```diff
@@ -8,27 +8,32 @@
 from app.models import Events, FeaturedStories
 
 
-def create_featured_story(story, left_right, quote):
+def create_featured_story(story, left_right, title, description, rank):
     """
     A utility function to create a featured story.
     :param story: the story object you would like to create in Featured Story
     :param left_right: left or right side where the image will go
-    :param quote: a famous quote that was once said by this activist
+    :param description: description of the activist
+    :param rank: the order of the featured story
+
     :return: None
     """
-    #
-    current_featured_story = FeaturedStories.query.filter_by(is_visible=True).one_or_none()
-    if current_featured_story is not None:
-        update_object({"is_visible": False}, FeaturedStories, current_featured_story.id)
+
+    #current_featured_story = FeaturedStories.query.filter_by(is_visible=True)
+    #if current_featured_story is not None:
+    #    update_object({"is_visible": False}, FeaturedStories, current_featured_story.id)
 
     featured_story = FeaturedStories(
         story_id=story.id,
         left_right=True if left_right == 'left' else False,
         is_visible=True,
-        quote=quote,
+        title=title,
+        description=description,
+        rank=rank
     )
 
     create_object(featured_story)
+    update_rank(featured_story.story_id, None, rank)
```

Comment:
Why is this needed upon creating the featured story?

Reply:
When a new featured story is added, it appears first on the carousel by default. The admin can  also decide its rank so the ranks in the database need to be updated as well.

Reply:
👍